ingestion-edge/bin/test

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -eox pipefail
+
+py.test \
+    --flake8 \
+    --docstyle \
+    --mypy \
+    --mypy-ignore-missing-imports \
+    --cov=./
```

Comment:
I think we should leave these args in docker-compose, and here use `"$@"` so that `docker-compose run --no-deps --rm test tests` continues to work as expected

Reply:
Done.